### ISSUE TITLE
Scrum points list randomly changing after saving

### ISSUE THREAD

**Issue by zerkalica** [256MbTeam/Redmine-Scrumbler#67]
Steps to reproduce:
1. Go to admin/scrumbler
2. Change List of possible values  to 0, 0.5, 1, 2, 3, default value: 1
3. Save
4. Reload page
5. List of possible values changing randomly after each page reload

After changing and saving List of possible values on admin/scrumbler this values changes randomly  on each page reload. Scrum poins in each task in backlog /projects/project/scrumbler_backlogs also changes randomy after page reload

http://i.imgur.com/Dlg6X.png

After page reload: http://i.imgur.com/EJzsY.png

**Comment by xeta** [comment]
I can't reproduce this bug. What is Redmine and Ruby version?

**Comment by zerkalica** [comment]
OS: debian squeeze
Redmine: 1.3.0 installed from rvm
Ruby: 1.8.7 installed from rvm

```
git clone git://github.com/256MbTeam/Redmine-Scrumbler.git vendor/plugins/redmine_scrumbler
rake db:migrate:plugin NAME=redmine_scrumbler
```

**Comment by xeta** [comment]
Does anyone still have this error?

**Comment by zerkalica** [comment]
Yes, bug still here. I try to dig redmine IssueCustomField api. May be  it's not a scrubmler bug.
